### ISSUE TITLE
Merge contacts

### ISSUE THREAD

**Issue by asbiin** [monicahq/monica#1687]
If 2 (or more) contacts are similar : same name, same email, etc. , they could be merged.

**Comment by asbiin** [comment]
Related to #613

**Comment by djaiss** [comment]
I don't think we should work on that as a priority. This is definitely a nice to have, but not critical in my opinion.

**Comment by mattdp** [comment]
+1 for this feature if you ever choose to revisit - any sort of API importation will probably result in dupes over time, e.g. if you import both list of FB friends and a list of LinkedIn contacts

**Comment by patrickduboucher** [comment]
I think this is a useful feature. Over time, people get added whatever means there either needs to be duplicate validation for manual entries, or a merge contact option after the fact.

**Comment by snachodog** [comment]
+1 for this feature

**Comment by enricomarogna** [comment]
+1 for this feature

**Comment by tomyvi** [comment]
+1 for me !

**Comment by FPMilan** [comment]
+100

**Comment by spcano01** [comment]
+1 for me - this is critical for any PRM/CRM/Contacts

**Comment by mwolfaardt** [comment]
+1

**Comment by virtadpt** [comment]
I could really use this right now.

**Comment by ohare93** [comment]
This feature is specifically useful for setting up Monica. When one imports 300 contacts from the other worse system, all the duplicates are going to come too. In order to start using Monica to it's fullest potential you need the duplicates resolved. Even if it is not all at once at the start, but when I search for a person to input an Activity we did together and I see two contacts for that person then the problems start: do I input my activity under one or the other? Should I try and manually merge them before continuing?

Just an out of the way "Merge" button next to Archive / Delete contact, where you can then choose one other contact and join all their info / activities / etc, would be great 👍

**Comment by fvalle1** [comment]
I create [this script](https://gist.github.com/fvalle1/9fc737bd9b0eab61a23d7724ecc675b5) to drop duplicates, if it could be useful to anyone

**Comment by digitaljdr** [comment]
checking in 2025 and this basic but key feature is still missing
